source/Ship.h

```diff
@@ -209,6 +209,7 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Set the commands for this ship to follow this timestep.
 	void SetCommands(const Command &command);
 	void SetCommands(const FireCommand &firingCommand);
```

Comment:
Turns out SetCommands with just FireCommand is never used alone, so we can just remove it.